src/discussions/discussions-home/DiscussionSidebar.test.jsx

```diff
@@ -100,8 +99,8 @@ describe('DiscussionSidebar', () => {
         },
       })]);
     renderComponent();
-    await act(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
+    waitFor(async () => expect(await screen.findAllByText('Thread by other users')).toBeTruthy());
```

Comment:
Using `waitFor` without `await` can cause tests to pass when they shouldn't. I tested this and it still passes with `await` before `waitFor` on this line, but it can be simplified to

```suggestion
    await screen.findAllByText('Thread by other users');
```

for the same reason as L87.